bundles/org.openhab.core.io.console/src/main/java/org/openhab/core/io/console/internal/extension/UserConsoleCommandExtension.java

```diff
@@ -12,9 +12,9 @@
  */
 package org.openhab.core.io.console.internal.extension;
 
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
Please import individual classes instead of complete packages using `*`.

> Rationale: Importing all classes from a package or static members from a class leads to tight coupling between packages or classes and might lead to problems when a new version of a library introduces name clashes.